detection_rules/main.py

```diff
@@ -235,16 +235,17 @@ def search_rules(query, columns, language, count, verbose=True, rules: Dict[str,
     rules = rules or {str(rule.path): rule for rule in RuleCollection.default()}
 
     for file_name, rule_doc in rules.items():
+        contents = rule_doc.contents.to_dict()
```

Comment:
for better consistency with type hinting I have this change in another PR: https://github.com/elastic/detection-rules/pull/1073/files#diff-ffa510a5b45c03c6d8104d48fade4c7a0039d0d8f4c7712eb808d15df6aadd65R270

Reply:
want to remove this update in favor of the linked changes?